src/App.tsx

```diff
@@ -1,28 +1,48 @@
-import { useState } from "react";
+import { Box, Divider } from "@mui/material";
 import "./App.css";
+import { SocialMediaIcon } from "Components/Icon";
+import { Typography } from "Components/Typography";
+import { TextField } from "Components/TextField";
+import { CheckboxText } from "Components/CheckboxText";
+import { TextLink } from "Components/TextLink";
 import { Button } from "Components/Button";
+import { SocialMediaButton } from "Components/Button";
 
 function App() {
-  const [count, setCount] = useState(0);
-
   return (
     <>
-      <h1>React</h1>
+      <Box sx={{ display: "flex", alignItems: "center" }}>
```

Comment:
This is login related.
Create a new folder named Pages and move this code under Login.tsx and import it here

Reply:
I just created the components there for testing, I am currently working on that . You can ignore this for now.